spec/models/cache_spec.rb

```diff
@@ -196,5 +196,81 @@ def lock_down_maximum
         Arturo::Feature.to_feature(@feature.symbol)
       }.to_not raise_error
     end
+
+    describe 'database errors' do
+      before do
+        Arturo::Feature.to_feature(@feature.symbol)
+        @feature.touch
+        Timecop.travel(Time.now + Arturo::Feature.cache_ttl + 5.seconds)
```

Comment:
Is time already frozen? I don't see it in the highest level before block.

Reply:
It wasn't frozen before, but we need it to travel to any point after TTL to force a refresh.